### ISSUE TITLE
code

### ISSUE THREAD

**Issue by ALLISWELL8** [JiaruiFeng/TAGLAS#10]
Hello, how can I obtain text information for nodes such as BBBP and Bace datasets

**Comment by JiaruiFeng** [comment]
Hi,

There is a bug in getting data samples for molecule datasets. We just fix it. Would you mind updating your version and using the following code to obtain text information for nodes in the molecule dataset:
```from TAGLAS import get_dataset
bbbp = get_dataset("bbbp")
# node text
bbbp[0].x
# edge text
bbbp[0].edge_attr
```

Sorry for the inconvenience and please let me know if you still face any issues.

Best.

**Comment by ALLISWELL8** [comment]
Thank you very much for your reply. I have another question, which is that when I obtained a dataset of Bace or other chemical types, I checked all the contents of the dataset and couldn't find the corresponding smiles format for the chemical molecules. Can you provide this? Thank you,

**Comment by JiaruiFeng** [comment]
Sorry for the late reply. Currently, we don't provide smiles for molecule datasets. We will add it in the near future. Here I provide pointers to where we convert smiles to graph format: [for molproperties](https://github.com/JiaruiFeng/TAGLAS/blob/main/datasets/chemmol/gen_data.py#L106), [for other datasets](https://github.com/JiaruiFeng/TAGLAS/blob/main/datasets/chemmol/gen_data.py#L156). 

Hope this could help!

Best.

**Comment by ALLISWELL8** [comment]
Thank you very much for your reply. I actually wanted to use the bace dataset here to train a GNN model, but I found that the structure of the bace dataset here is different from the structure of the bace dataset I have seen before. Then I used word2vec to directly convert the text in bace. x into vectors, and then trained the GNN model. I found that the effect was too poor. The reason I currently think of is that the vectors generated by the word2vec model are not good, which leads to poor performance. I would like to ask how you used it to train the GNN model? Thank you very much for your reply. 
Best.
